CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## [2.19.1] - 2019-11-17
+
+## Added
+- Support for eirini SSH feature
+
+## Changed
+- Bumped cflinuxfs3 release to 0.141.0
+- Bumped eirini release to 0.0.23
+- Bumped go-buildpack release to 1.9.2
+- Bumped nodejs-buildpack release to 1.7.1
+- Bumped java-buildpack release to 4.24.0
+- Bumped SLE12 and SLE15 stack
+ 
+## Fixed
+- Fixed apiVersion in Chart yaml(s) to point to Helm API version (v1)
+- Turned binlog on for pxc config to enable transaction recovery
+
+## [2.19.0] - 2019-10-17
+
+## Added
+- Support for external CF-related databases, including CCDB & UAA when embedded in SCF
+- Support for eirini to run on GKE and EKS
+- Support for eirini to run on CRI-O-based Kubernetes environments
+- Ingress controller now available for UAA embedded in SCF
+- AUDIT_WRITE capabilities added for CRI-O
+
+## Changed
+- Eirini will use cflinuxfs3 as its default stack
```

Comment:
s/cflinuxfs3/sle15 -> https://github.com/SUSE/scf/blob/rc/container-host-files/etc/scf/config/scripts/override-default-stack.sh#L5

Reply:
Fixed in c634c36076440012ca134a9d4bdac25ab193a6ba